### PR TITLE
13.0 mig hr job category

### PR DESCRIPTION
**Pull request by meodoo**

### CODE REVIEW

File: hr_job_category/models/hr.py [hunk 1]
```diff
@@ -1,76 +1,75 @@
...
     )
 
 
 class HRContract(models.Model):
-    _inherit = 'hr.contract'
+    _inherit = "hr.contract"
 
-    @api.multi
     def _remove_tags(self, employee_id=None, job_id=None):
         # TODO write tags only once
```
Comment (jarroyomorales) [inline hr_job_category/models/hr.py:27]: Another way to do this function would be to keep a list of the ids you want to keep and then perform a [(6, 0, [ids])] write.
This way you would only do one write but if don't know if it is more efficient or not. If we want to keep it like this we can remove the TODO

File: hr_job_category/models/hr.py [hunk 2]
```diff
@@ -1,76 +1,75 @@
...
         if not employee_id or not job_id:
             return
-        employee = self.env['hr.employee'].browse(employee_id)
+        employee = self.env["hr.employee"].browse(employee_id)
         empl_tags = employee.category_ids
-        job = self.env['hr.job'].browse(job_id)
-        _logger.debug('Removing employee tags if tag exists on contract '
-                      'job: %s', empl_tags)
+        job = self.env["hr.job"].browse(job_id)
+        _logger.debug(
```
Comment (jarroyomorales) [inline hr_job_category/models/hr.py:33]: Is this logger necessary?

File: hr_job_category/models/hr.py [hunk 3]
```diff
@@ -1,76 +1,75 @@
...
+        employee = self.env["hr.employee"].browse(employee_id)
         empl_tags = employee.category_ids
-        job = self.env['hr.job'].browse(job_id)
+        job = self.env["hr.job"].browse(job_id)
         for tag in job.category_ids:
             if tag not in empl_tags:
-                _logger.debug("Adding employee tag if job tag doesn't "
-                              "exists: %s", tag.name)
-                employee.write({'category_ids': [(4, tag.id)]})
+                _logger.debug(
```
Comment (jarroyomorales) [inline hr_job_category/models/hr.py:48]: Same

File: hr_job_category/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -3,4 +3,4 @@ to support tagging employees based on their job positions.
 For example, all Supervisors could be attached to the Supervisors category.
 Define which categories a job belongs to in the configuration for the job.
 When an employee is assigned a particular job the categories attached to that
-job will be attached to the employee record as well.
\ No newline at end of file
+job will be attached to the employee record as well.
```
Comment (jarroyomorales) [inline hr_job_category/readme/DESCRIPTION.rst:6]: I cant see what changed here

**Review by gurneyalex — APPROVED** [review]

**Review by ovnicraft — APPROVED** [review]

**Comment by Saran440** [comment]
@Murtaza-Mithaiwala Hi! Could you pre-commit this module?

**Comment by leemannd** [comment]
Has been merged in -> https://github.com/OCA/hr/pull/895
Can be closed

**Comment by Saran440** [comment]
merged #895

**Pull request closed without merging**